### ISSUE TITLE
Fails after 1st run (Win7, v2.0.2)

### ISSUE THREAD

**Issue by davidhund** [paulirish/pwmetrics#121]
Running `pwmetrics http://localhost:3000/ --runs=3` fails after the first run.

### Steps to reproduce.
`pwmetrics http://localhost:3000/ --runs=3`

### Stack trace
```
Launching Chrome
  ✓ Success: Run 1 of 3 finished successfully.
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 127.0.0.1:9222
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1090:14)
```

### Environment
1. `pwmetrics` version: 2.0.2
2. Chrome version: 58.0.3029.110 (64-bit)
3. OS version: Win 7

I got similar errors on non-localhost sites. Doing just 1 run is fine.

Also: I've noticed the Chrome Translation Dialog pops up, possibly interfering with Lighthouse?
[Current Lighthouse version supports `--disable-translate`](https://github.com/GoogleChrome/lighthouse/issues/896) but adding that flag does not seem to work. I assume support will be added with #119 ?

**Comment by pedro93** [comment]
Hello, thanks for the filling the issue.

Reproducing localhost wouldn't help without having the source code of what you serve nor the method by which you serve it. For instance if your http server is a node process, to run pwmetrics it has to be on a separate process.

Could you provide an example of a non-localhost site where I can reproduce this? 

Support for lighthouse v2 flags will be added with https://github.com/paulirish/pwmetrics/issues/119 yes

**Comment by davidhund** [comment]
Thanks for your quick reply @pedro93!

Just tested with `pwmetrics https://google.com/ --runs=3`:

```
Launching Chrome
  ✓ Success: Run 1 of 3 finished successfully.
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 127.0.0.1:9222
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1090:14)
```

As for localhost: I am serving it through browserSync which proxies a local Apache. 
Running pwmetrics directly on the local Apache url fails too b.t.w.

**Comment by pedro93** [comment]
That is very strange, I just ran pwmetrics on my machine (ubuntu) and it worked:

![pwmetrics](https://cloud.githubusercontent.com/assets/3718654/26410859/53f27e48-409c-11e7-9ad9-c77cb6472cdb.png)

The only variation I see is the operating system. I have a windows 8 machine at home, I will try there in a couple of hours. 

In the meantime could you see in the lighthouse repo if there are windows-related issues opened? Also could try to run lighthouse v2 from the command line and see if you get current output? This could be a windows-specific 1.6.5 lighthouse bug, if thats the case https://github.com/paulirish/pwmetrics/issues/119 will solve the problem when landed..

**Comment by denar90** [comment]
Hey guys. I assume that this problem can be present on windows and it could be related to chrome launcher. But it should work after updating to LH v2. I'll test current version and with LH v2 on windows later today. Will see how it goes.

**Comment by denar90** [comment]
hmm, I'm facing kinda similar errors even with LH v2. I used virtualbox and windows10, so it was painful
![image](https://cloud.githubusercontent.com/assets/6231516/26426067/bf305764-40df-11e7-83bb-12f1afc32c75.png)

**Comment by samccone** [comment]
it seems related to this eh...
https://github.com/GoogleChrome/lighthouse/issues/2337

**Comment by pedro93** [comment]
@samccone FYI pwmetrics does not use lighthouse v2 yet. 

That is in a PR currently. Does the issue https://github.com/GoogleChrome/lighthouse/issues/2337 also happens for lighthouse v1.6.5? If not this could be something else altogether.

We use a commit specific version: paulirish/lighthouse#c74ea959"

**Comment by davidhund** [comment]
@samccone @pedro93 — thanks for looking into this! 

I assume — as https://github.com/GoogleChrome/lighthouse/pull/2359 fixes the related LH v2.0.0 issue —  we can expect this to be coming in with #119?

**Comment by pedro93** [comment]
Yes we intend to merge https://github.com/paulirish/pwmetrics/issues/119 as soon https://github.com/GoogleChrome/lighthouse/issues/2251 is landed and published.

**Comment by samccone** [comment]
The root issues was pre lighthouse 2 :)

**Comment by denar90** [comment]
Should be fixed by #120 
Feel free to reopen.
